engine/transaction/record/record_outline_test.go

```diff
@@ -278,46 +259,78 @@ func TestRecordOutlineOpReturnErrorCases(t *testing.T) {
 			},
 			expectErr: txerrors.ErrAnnotationMismatch,
 		},
-		"error during broadcasting": {
-			broadcaster: newMockBroadcaster().withError(errors.New("broadcast error")),
-			repo: newMockRepository().withOutput(database.Output{
-				TxID: givenTXWithOpReturn(t).InputUTXO(0).TxID,
-				Vout: givenTXWithOpReturn(t).InputUTXO(0).Vout,
-			}),
-			outline: &outlines.Transaction{
-				BEEF: givenTXWithOpReturn(t).BEEF(),
-				Annotations: transaction.Annotations{
-					Outputs: transaction.OutputsAnnotations{
-						0: &transaction.OutputAnnotation{
-							Bucket: bucket.Data,
-						},
-					},
-				},
-			},
-			expectErr: txerrors.ErrTxBroadcast,
-		},
 	}
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
 			// given:
-			service := record.NewService(tester.Logger(t), test.repo, test.broadcaster)
-			initialOutputs := test.repo.getAllOutputs()
-			initialData := test.repo.getAllData()
+			given, then := testabilities.New(t)
+			service := given.WithStoredOutputs(test.storedOutputs...).NewRecordService()
```

Comment:
I would rather make it separated calls like this:
```suggestion
			given.WithStoredOutputs(test.storedOutputs...)
			
			service := given.NewRecordService()
```

Reply:
Adjusted 👍